platform/ios/src/MGLAPIClient.m

```diff
@@ -87,7 +87,7 @@ - (NSURLRequest *)requestForEvents:(NS_ARRAY_OF(MGLMapboxEventAttributes *) *)ev
 }
 
 - (void)setupBaseURL {
-    NSString *testServerURL = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"MGLMetricsTestServerURL"];
+    NSString *testServerURL = [[NSUserDefaults standardUserDefaults] stringForKey:@"MGLMapboxMetricsTestServerURL"];
```

Comment:
Ah, I guess that’s probably too much hassle, then. Sorry about that. It turns out that you can use `-URLForKey:` and `-setURL:forKey:` without going through NSData, but you have to wrap in NSData to register a default.